epoxy-adapter/src/main/java/com/airbnb/epoxy/EpoxyModel.java

```diff
@@ -287,11 +287,11 @@ public void onViewDetachedFromWindow(T view) {
 
   @Override
   public String toString() {
-    return getClass().getSimpleName() + "{" +
-        "id=" + id +
-        ", layout=" + getLayout() +
-        ", shown=" + shown +
-        ", addedToAdapter=" + addedToAdapter +
-        '}';
+    return getClass().getSimpleName() + "{"
+        + "id=" + id
```

Comment:
Is this the correct Airbnb java open source formatting? I can start using this in Lottie if that's the case.